api/python/cell_census/src/cell_census/open.py

```diff
@@ -5,12 +5,11 @@
 import s3fs
 import tiledb
 import tiledbsoma as soma
+from tiledbsoma.options import SOMATileDBContext
```

Comment:
See above comment about the unqualified import of names.  E.g.,
> https://google.github.io/styleguide/pyguide.html#22-imports

Reply:
I think we need to export the `tiledbsoma.options` sub-pkg exports as a top-level `tiledbsoma` export?


Usage such as this:
```
import tiledbsoma as soma
...
soma.SOMATileDBContext()
```
seems better than just having an "unaffiliated" `options` package alias:
```
from tiledbsoma import options

options.SOMATileDBContext()
```.

Though this could work:
```
from tiledbsoma import options as soma_options

soma_options.SOMATileDBContext()
```
Noted in https://github.com/single-cell-data/TileDB-SOMA/issues/791

Reply:
> I think we need to export the tiledbsoma.options sub-pkg exports as a top-level tiledbsoma export?

No, it should just work as is.  I.e.,
```
In [5]: import tiledbsoma as soma

In [6]: soma.options.SOMATileDBContext
Out[6]: tiledbsoma.options.soma_tiledb_context.SOMATileDBContext
```

Reply:
Indeed, and PyCharm has misled me--it complains about `soma.options.SOMATileDBContext`. 🤔